crates/driver/src/infra/api/routes/notify/dto/notify_request.rs

```diff
@@ -0,0 +1,25 @@
+use {crate::infra::notify, serde::Deserialize, serde_with::serde_as};
+
+#[serde_as]
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum NotifyRequest {
+    /// The driver won multiple consecutive auctions but never settled them.
+    UnsettledConsecutiveAuctions,
+}
+
+impl From<NotifyRequest> for notify::Kind {
+    fn from(value: NotifyRequest) -> Self {
+        match value {
+            NotifyRequest::UnsettledConsecutiveAuctions => {
+                notify::Kind::Banned(notify::BanReason::UnsettledConsecutiveAuctions)
```

Comment:
nit improvement proposal: As a solver I would be interested in knowing the block number when i will be enabled again.